charts/base-cluster/templates/weave/credentials.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.weaveworks.enabled }}
+  {{ $pwd := include "common.secrets.passwords.manage" (dict "secret" "cluster-user-auth" "key" "password-dec" "providedValues" (list "weaveworks.adminUser.password") "length" 20 "strong" false "skipB64enc" true "skipQuote" true "context" (dict "Values" .Values "Release" ((dict "IsUpgrade" false "IsInstall" true "Namespace" "flux-system") | mergeOverwrite (deepCopy .Release)))) }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster-user-auth
+  namespace: flux-system
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: gitops-dashboard
+stringData:
+  password: {{ $pwd | bcrypt | quote }}
+  username: {{ .Values.weaveworks.adminUser.username | quote }}
+  password-dec: {{ $pwd | quote }}
```

Comment:
Currently you can not update the password afterwards. If you change the password in the values file after an initial installation, it won't update

Reply:
fixed but the password which is printed in the notes is now wrong